app/Tables.hs

```diff
@@ -100,16 +100,16 @@ baseNumbers =
   [ (1, "ichi")
   , (2, "ni")
   , (3, "san")
-  , (4, "shi") -- yon
+  , (4, "yon")
```

Comment:
Maybe we should start thinking about how to accept both representations in this case. I'm learning it as "yon" but someone else might be using "shi". Maybe open a separate issue as feature request for this?

Reply:
Will do!